Src/Microsoft.Scripting/Microsoft.Scripting.csproj

```diff
@@ -11,133 +11,9 @@
     <BaseAddress>857735168</BaseAddress>
     <CodeAnalysisRuleSet>$(AssemblyName).ruleset</CodeAnalysisRuleSet>
     <DocumentationFile>$(OutputPath)\$(AssemblyName).xml</DocumentationFile>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'FxCop|AnyCPU' ">
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'v4Debug|AnyCPU' ">
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'v4Release|AnyCPU' ">
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-  </PropertyGroup>
+  </PropertyGroup>  
   <ItemGroup>
-    <Compile Include="..\AssemblyInfo.cs">
```

Comment:
I think we'll want to keep this one (here and also in the other csproj files). I probably inadvertently removed it from my original PR. Note that you make have to change the Link to something else (e.g. Properties\AssemblyInfo.Generated.cs) if it causes conflicts with an existing AssemblyInfo.cs file.